pkg/kubectl/cmd/cp_test.go

```diff
@@ -5,7 +5,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-    http://www.apache.org/licenses/LICENSE-2.0
+		http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
the indent on this line seems incorrect.
have a look this for ref:
https://github.com/mgugino-upstream-stage/kubernetes/blob/master/pkg/kubectl/cmd/apiversions.go

might as well bump the year to 2018, while at it. :)